__tests__/apps.test.js

```diff
@@ -0,0 +1,34 @@
+const app = require("../apps.js");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+  //seeding the db with the test data
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("Get /api/topics", () => {
+  test("status:200, responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
```

Comment:
would be nice to have an error handling test for an incorrect path being entered (e.g /api/topic) but this is not essential right now!